src/utilities/codemirror/codemirror-utils.test.ts

```diff
@@ -117,3 +119,17 @@ describe('isHexValue', function () {
     expect(isHexValue('0x12xx')).toBe(false);
   });
 });
+describe('getDefaultVariableArgs', function () {
```

Comment:
Not super important, but could this be an arrow function instead to be consistent? I just noticed that the above tests also have anonymous functions instead of arrow functions as well, but I'll leave that up to you if you want to fix that here or another effort.

Reply:
Addressed with 2bbf7724e6e3bd21582e809fa790dd652d6158e0